src/units/abilities.cpp

```diff
@@ -980,7 +1035,7 @@ std::string attack_type::weapon_specials_value(const std::set<std::string>& chec
 	for(const auto [key, cfg] : specials_.all_children_view()) {
 		if((checking_tags.count(key) != 0)){
```

Comment:
double parentheses.
`
```suggestion
		if(checking_tags.count(key) != 0) {
```

Reply:
done